### ISSUE TITLE
t/winxed/001_load.t .... not ok (Test exited prematurely)

### ISSUE THREAD

**Issue by leto** [letolabs/parrot-libgit2#20]
There is a bug in Rosella or our test harness which exits with code 0, but all the tests are not passing. The latest "passing" bulid on Travis is actually failing.

**Comment by sa1** [comment]
Yes, I am aware of it. All the tests pass on my system. I need to figure out which test is causing that. I may need to split the tests into different files. I suspect it is because of the clone test, which tries to clone a repository in /tmp.

**Comment by leto** [comment]
One reason is that libgit2 was not being installed on by Travis. I just pushed a commit that hopefully fixes that, although it uses the master branch of libgit2, which might cause issues.
